services/movie.py

```diff
@@ -28,13 +33,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Consider using `list[int]` for the `genres_ids` and `actors_ids` parameters to be consistent with the type annotations in the `get_movies` function.